### ISSUE TITLE
Review alt text for images on the pages

### ISSUE THREAD

**Issue by ryelle** [WordPress/wporg-main-2022#50]
Currently no image have alt text (I think), but they should. This needs to be written.

**Comment by thetinyl** [comment]
@ryelle Alt text has now been added for all images. Listing them below by section just to double-check I haven't missed any.

**Homepage**

- Powerful and empowering section: sample styles image
- Millions of possibilities section: company logos and showcase gallery image
- Community section: WCEU group photograph
- What's new in WP section: social media icons (just Twitter and FB, will the others be added before launch?)

**Download**

- Inspiration strikes section: updated alt text for app download buttons

**Comment by iandunn** [comment]
Aside: IIRC, there was some discussion a few years ago about Gutenberg requiring `alt` text to be entered when publishing/updating. That was declined b/c it'd be too heavy-handed for the entire userbase, but it may be something we want to do on w.org. It's easy for folks to forget or not know that they should add them.

**Comment by ryelle** [comment]
These are great, thanks @thetinyl 

> requiring alt text to be entered when publishing/updating.

There are still some cases where there are decorative images (like the background image of "Dream it, build it") which should be empty. I don't think we need to do anything about this right now, but that could be added in the future.
